lib/uniquable.rb

```diff
@@ -0,0 +1,9 @@
+module Uniquable
+  def self.unique_seasons_hash(data)
+    game_by_season = data[:games]
+    total_season = (game_by_season.uniq { |season| season[:season] }).map { |season| season[:season] }
+    total_count = Hash.new
+    total_season.each { |season| total_count.store(season, []) }
+    return game_by_season, total_count
```

Comment:
I will need some clarity on how this return works

Reply:
Also same. I thought I understood the first time you explained it to us, oops.